.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,12 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
```

Comment:
Maybe change the name to Flix Standard or something? Same for the file name.

Reply:
Am I allowed to? I don't know which names have significance.

Reply:
Changing the value here and file name just does this:

![image](https://github.com/user-attachments/assets/2a649fc4-bf0d-4638-9c2f-4c38ee9eca62)

Reply:
Can do it next. Let us see what happens when we merge this.